software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakCluster.java

```diff
@@ -54,6 +55,16 @@
 
     AttributeSensor<String> NODE_LIST = Sensors.newStringSensor("riak.cluster.nodeList", "List of nodes (including ports), comma separated");
 
+    @SetFromFlag("search")
+    ConfigKey<Boolean> SEARCH = ConfigKeys.newBooleanConfigKey(
+            "riak.search", "Deploy Solr and configure Riak to use it", false);
+
+    @SetFromFlag("memberSpec")
+    ConfigKey<EntitySpec<?>> SOLR_MEMBER_SPEC = ConfigKeys.newConfigKey(
+            new TypeToken<EntitySpec<?>>() {
+            },
+            "riak.solr.memberspec", "Solr entity spec for creating new cluster members", null);
```

Comment:
No need to set null as the default value (this is the value used if no explicit one is specified).